startup/49-utils.py

```diff
@@ -103,34 +104,83 @@ def printfig():
     os.system("lp -d HXN-printer-1 /home/xf05id1/tmp/temp.png")
 
 
-def estimate_scan_duration(xnum, ynum, dwell, scantype=None, event_delay=None):
+def print_warning_message(msg):
+    msg_len = len(msg) + 2
+    print(f"\n{'*' * msg_len}")
+    print(f' {msg} ')
+    print(f"{'*' * msg_len}\n")
```

Comment:
maybe I need another example. the documentation I am seeing is will break lines and do that sort of text wrapping, not adding borders. This could always be added as a separate commit